### ISSUE TITLE
Remove the `digest` attribute under `mode`, which may be more conducive to user configuration

### ISSUE THREAD

**Issue by linghengqian** [apache/shardingsphere#13693]
## Feature Request

**For English only**, other languages will not accept.

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot make decision by current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Is your feature request related to a problem?
No.
### Describe the feature you would like.

In the Slack group of ShardingSphere, Haoran Meng once gave a method to configure the `ACL` information of `zookeeper` in the `shardingsphere-proxy`.
![image](https://user-images.githubusercontent.com/20187731/142432916-dc90055c-0efc-4d48-a30d-0c311a02550f.png)
![image](https://user-images.githubusercontent.com/20187731/142432947-cdeb80e8-a137-4bd4-9c53-d0aa8cfb9cb3.png)

Under the `props` tag of the `mode` tag, there is an attribute that is not mentioned in the official website document as `digest`. 
![image](https://user-images.githubusercontent.com/20187731/142435913-03b4f930-0542-4106-9d42-dcb19f276046.png)

Through this attribute, you can configure the `id` part in `[scheme:id:permissions]` to complete the digest Authentication, but observing the source code of `shardingsphere`, it can be seen that this is only limited to `digest` ACL verification.
Take an example as a lead.
![image](https://user-images.githubusercontent.com/20187731/142433252-97e14632-2ad5-4e74-b628-17310af46297.png)
Through the `org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository` and `org.apache.shardingsphere.mode.repository.cluster.zookeeper.props.ZookeeperPropertyKey` these two classes can verify the existence of the `digest` property .
![image](https://user-images.githubusercontent.com/20187731/142434356-9843e9f4-519f-48ef-9d76-63b2c9c6af55.png)
![image](https://user-images.githubusercontent.com/20187731/142434431-54e3cf0a-1821-4048-840c-f1eb4eb7c939.png)

It is not difficult to know that the nodes in `server-lists` will be completely resolved into `zookeeper` addresses.
Through the ideas given in https://github.com/apache/dubbo-admin/issues/848, it is completely possible to configure the `ACL` information through the `zookeeper url` instead of the `digest` attribute. At the same time, `org.apache.shardingsphere.mode.repository.cluster.etcd.EtcdRepository` does not give the special configuration of `basicAuth` properties. It seems that the configuration of `zookeeper` as shown in the figure below is completely fine.
![image](https://user-images.githubusercontent.com/20187731/142435357-f3f3f5ac-9018-44d9-9c16-2e781b32a5d7.png)
![image](https://user-images.githubusercontent.com/20187731/142435647-4779dbca-407e-4a14-92a6-536816b997e2.png)

In summary, I put forward my point-removing the `digest` attribute under the `mode` tag will make it easier for users who have been using zookeeper to understand the configuration.

**Comment by menghaoranss** [comment]
@linghengqian Thank for your feedback, does it work now when you configure the permissions in `server-lists`?

**Comment by linghengqian** [comment]
> @linghengqian Thank for your feedback, does it work now when you configure the permissions in `server-lists`?

Yes, it is normal to configure `ACL` information in the `server-lists`.

**Comment by linghengqian** [comment]
@menghaoranss 
I'm sorry to raise such an issue... I found that I couldn't reproduce this operation in `server-lists` on another machine, and the error in `input string` was also prompted. I realized that it might be because of the old machine,the `volumes` was not cleaned up, which caused him to connect to `zookeeper` normally. In fact, the encapsulation of `curator` by `shardingsphere` does not support writing parameters in `zookeeper url`, which is my problem. I will close this issue .

![image](https://user-images.githubusercontent.com/20187731/143037711-81cd1d4a-ce76-4855-af53-ceea4dd02a36.png)


```shell
Starting the ShardingSphere-Proxy ...

The port is 3308

The classpath is /opt/shardingsphere-proxy/conf:.:/opt/shardingsphere-proxy/lib/*:/opt/shardingsphere-proxy/ext-lib/*

Please check the STDOUT file: /opt/shardingsphere-proxy/logs/stdout.log

	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)

	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:716)

	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:342)

	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1262)

[INFO ] 2021-11-23 21:46:49.914 [main-SendThread(node1.zookeeper.lingh.com:2181)] org.apache.zookeeper.ClientCnxn - Opening socket connection to server node1.zookeeper.lingh.com/192.168.240.3:2181.

[INFO ] 2021-11-23 21:46:49.914 [main-SendThread(node1.zookeeper.lingh.com:2181)] org.apache.zookeeper.ClientCnxn - SASL config status: Will not attempt to authenticate using SASL (unknown error)

[INFO ] 2021-11-23 21:46:49.914 [main-SendThread(node1.zookeeper.lingh.com:2181)] org.apache.zookeeper.ClientCnxn - Socket connection established, initiating session, client: /192.168.240.2:35068, server: node1.zookeeper.lingh.com/192.168.240.3:2181

[INFO ] 2021-11-23 21:46:49.959 [main-SendThread(node1.zookeeper.lingh.com:2181)] org.apache.zookeeper.ClientCnxn - Session establishment complete on server node1.zookeeper.lingh.com/192.168.240.3:2181, session id = 0x100a1f91def000e, negotiated timeout = 40000

[INFO ] 2021-11-23 21:46:49.960 [main-EventThread] o.a.c.f.state.ConnectionStateManager - State change: RECONNECTED

[INFO ] 2021-11-23 21:46:50.000 [main-EventThread] o.a.c.framework.imps.EnsembleTracker - New config event received: {}

[INFO ] 2021-11-23 21:47:00.982 [main] o.a.c.f.imps.CuratorFrameworkImpl - Starting

[INFO ] 2021-11-23 21:47:01.046 [main] org.apache.zookeeper.ZooKeeper - Client environment:zookeeper.version=3.6.0--b4c89dc7f6083829e18fae6e446907ae0b1f22d7, built on 02/25/2020 14:38 GMT

[INFO ] 2021-11-23 21:47:01.047 [main] org.apache.zookeeper.ZooKeeper - Client environment:host.name=21e60dd2c586

[INFO ] 2021-11-23 21:47:01.047 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.version=1.8.0_312

[INFO ] 2021-11-23 21:47:01.047 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.vendor=Oracle Corporation

[INFO ] 2021-11-23 21:47:01.047 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.home=/usr/local/openjdk-8/jre

[INFO ] 2021-11-23 21:47:01.047 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.class.path=/opt/shardingsphere-proxy/conf:.:/opt/shardingsphere-proxy/lib/HikariCP-3.4.2.jar:/opt/shardingsphere-proxy/lib/accessors-smart-1.2.jar:/opt/shardingsphere-proxy/lib/aggdesigner-algorithm-6.0.jar:/opt/shardingsphere-proxy/lib/animal-sniffer-annotations-1.18.jar:/opt/shardingsphere-proxy/lib/antlr4-runtime-4.9.2.jar:/opt/shardingsphere-proxy/lib/apiguardian-api-1.1.0.jar:/opt/shardingsphere-proxy/lib/asm-5.0.4.jar:/opt/shardingsphere-proxy/lib/atomikos-util-4.0.6.jar:/opt/shardingsphere-proxy/lib/audience-annotations-0.5.0.jar:/opt/shardingsphere-proxy/lib/avatica-core-1.18.0.jar:/opt/shardingsphere-proxy/lib/avatica-metrics-1.18.0.jar:/opt/shardingsphere-proxy/lib/calcite-core-1.27.0.jar:/opt/shardingsphere-proxy/lib/calcite-linq4j-1.27.0.jar:/opt/shardingsphere-proxy/lib/checker-qual-2.11.1.jar:/opt/shardingsphere-proxy/lib/commons-codec-1.10.jar:/opt/shardingsphere-proxy/lib/commons-collections4-4.2.jar:/opt/shardingsphere-proxy/lib/commons-compiler-3.0.11.jar:/opt/shardingsphere-proxy/lib/commons-exec-1.3.jar:/opt/shardingsphere-proxy/lib/commons-io-2.8.0.jar:/opt/shardingsphere-proxy/lib/commons-lang-2.6.jar:/opt/shardingsphere-proxy/lib/commons-lang3-3.4.jar:/opt/shardingsphere-proxy/lib/commons-logging-1.1.3.jar:/opt/shardingsphere-proxy/lib/curator-client-5.1.0.jar:/opt/shardingsphere-proxy/lib/curator-framework-5.1.0.jar:/opt/shardingsphere-proxy/lib/curator-recipes-5.1.0.jar:/opt/shardingsphere-proxy/lib/elasticjob-api-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-dataflow-executor-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-error-handler-general-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-error-handler-spi-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-executor-kernel-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-http-executor-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-infra-common-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-lite-core-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-lite-lifecycle-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-registry-center-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-script-executor-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-simple-executor-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-tracing-api-3.0.1.jar:/opt/shardingsphere-proxy/lib/elasticjob-tracing-rdb-3.0.1.jar:/opt/shardingsphere-proxy/lib/error_prone_annotations-2.3.4.jar:/opt/shardingsphere-proxy/lib/esri-geometry-api-2.2.0.jar:/opt/shardingsphere-proxy/lib/failsafe-2.3.3.jar:/opt/shardingsphere-proxy/lib/failureaccess-1.0.1.jar:/opt/shardingsphere-proxy/lib/groovy-2.4.19-indy.jar:/opt/shardingsphere-proxy/lib/grpc-api-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-context-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-core-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-grpclb-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-netty-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-protobuf-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-protobuf-lite-1.27.1.jar:/opt/shardingsphere-proxy/lib/grpc-stub-1.27.1.jar:/opt/shardingsphere-proxy/lib/gson-2.8.6.jar:/opt/shardingsphere-proxy/lib/guava-29.0-jre.jar:/opt/shardingsphere-proxy/lib/httpclient-4.5.9.jar:/opt/shardingsphere-proxy/lib/httpcore-4.4.11.jar:/opt/shardingsphere-proxy/lib/jackson-annotations-2.10.0.jar:/opt/shardingsphere-proxy/lib/jackson-core-2.10.0.jar:/opt/shardingsphere-proxy/lib/jackson-databind-2.10.0.jar:/opt/shardingsphere-proxy/lib/jackson-dataformat-yaml-2.10.0.jar:/opt/shardingsphere-proxy/lib/janino-3.0.11.jar:/opt/shardingsphere-proxy/lib/jcl-over-slf4j-1.7.7.jar:/opt/shardingsphere-proxy/lib/jetcd-common-0.5.0.jar:/opt/shardingsphere-proxy/lib/jetcd-core-0.5.0.jar:/opt/shardingsphere-proxy/lib/jetcd-resolver-0.5.0.jar:/opt/shardingsphere-proxy/lib/json-path-2.4.0.jar:/opt/shardingsphere-proxy/lib/json-smart-2.3.jar:/opt/shardingsphere-proxy/lib/jsr305-3.0.2.jar:/opt/shardingsphere-proxy/lib/jta-1.1.jar:/opt/shardingsphere-proxy/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar:/opt/shardingsphere-proxy/lib/log4j-1.2.17.jar:/opt/shardingsphere-proxy/lib/logback-classic-1.2.0.jar:/opt/shardingsphere-proxy/lib/logback-core-1.2.0.jar:/opt/shardingsphere-proxy/lib/mchange-commons-java-0.2.15.jar:/opt/shardingsphere-proxy/lib/memory-0.9.0.jar:/opt/shardingsphere-proxy/lib/netty-buffer-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-codec-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-codec-http-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-codec-http2-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-codec-socks-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-common-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-handler-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-handler-proxy-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-resolver-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-transport-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-transport-native-epoll-4.1.69.Final-linux-aarch_64.jar:/opt/shardingsphere-proxy/lib/netty-transport-native-epoll-4.1.69.Final-linux-x86_64.jar:/opt/shardingsphere-proxy/lib/netty-transport-native-epoll-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/netty-transport-native-unix-common-4.1.69.Final.jar:/opt/shardingsphere-proxy/lib/perfmark-api-0.19.0.jar:/opt/shardingsphere-proxy/lib/postgresql-42.2.5.jar:/opt/shardingsphere-proxy/lib/proto-google-common-protos-1.17.0.jar:/opt/shardingsphere-proxy/lib/protobuf-java-3.11.0.jar:/opt/shardingsphere-proxy/lib/protobuf-java-util-3.11.0.jar:/opt/shardingsphere-proxy/lib/quartz-2.3.2.jar:/opt/shardingsphere-proxy/lib/shardingsphere-authority-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-authority-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-cluster-mode-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-cluster-mode-repository-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-cluster-mode-repository-etcd-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-cluster-mode-repository-zookeeper-curator-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-discovery-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-discovery-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-discovery-distsql-handler-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-discovery-distsql-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-discovery-distsql-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-discovery-mgr-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-protocol-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-protocol-mysql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-protocol-opengauss-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-db-protocol-postgresql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-distsql-parser-engine-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-distsql-parser-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-encrypt-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-encrypt-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-encrypt-distsql-handler-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-encrypt-distsql-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-encrypt-distsql-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-binder-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-common-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-context-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-datetime-spi-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-executor-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-merge-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-optimize-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-rewrite-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-infra-route-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-jdbc-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-memory-mode-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-mode-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-backend-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-bootstrap-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-frontend-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-frontend-mysql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-frontend-opengauss-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-frontend-postgresql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-proxy-frontend-spi-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-readwrite-splitting-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-readwrite-splitting-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-readwrite-splitting-distsql-handler-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-readwrite-splitting-distsql-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-readwrite-splitting-distsql-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-distsql-handler-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-distsql-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-distsql-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-mysql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-opengauss-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-scaling-postgresql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-shadow-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-shadow-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-shadow-distsql-handler-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-shadow-distsql-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-shadow-distsql-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sharding-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sharding-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sharding-distsql-handler-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sharding-distsql-parser-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sharding-distsql-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-single-table-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-single-table-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-spi-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-engine-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-mysql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-opengauss-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-oracle-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-postgresql-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-spi-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-sql92-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-sqlserver-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-sql-parser-statement-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-standalone-mode-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-standalone-mode-repository-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-standalone-mode-repository-file-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-system-datetime-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-transaction-api-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-transaction-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-transaction-xa-atomikos-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-transaction-xa-core-5.0.0.jar:/opt/shardingsphere-proxy/lib/shardingsphere-transaction-xa-spi-5.0.0.jar:/opt/shardingsphere-proxy/lib/sketches-core-0.9.0.jar:/opt/shardingsphere-proxy/lib/slf4j-api-1.7.7.jar:/opt/shardingsphere-proxy/lib/snakeyaml-1.16.jar:/opt/shardingsphere-proxy/lib/transactions-4.0.6.jar:/opt/shardingsphere-proxy/lib/transactions-api-4.0.6.jar:/opt/shardingsphere-proxy/lib/transactions-jdbc-4.0.6.jar:/opt/shardingsphere-proxy/lib/transactions-jta-4.0.6.jar:/opt/shardingsphere-proxy/lib/uzaygezen-core-0.2.jar:/opt/shardingsphere-proxy/lib/zookeeper-3.6.0.jar:/opt/shardingsphere-proxy/lib/zookeeper-jute-3.6.0.jar:/opt/shardingsphere-proxy/ext-lib/mysql-connector-java-8.0.27.jar:/opt/shardingsphere-proxy/ext-lib/lingh-public-common-1.1-beta.jar

[INFO ] 2021-11-23 21:47:01.047 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.library.path=/usr/java/packages/lib/amd64:/usr/lib64:/lib64:/lib:/usr/lib

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.io.tmpdir=/tmp

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:java.compiler=<NA>

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:os.name=Linux

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:os.arch=amd64

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:os.version=3.10.0-1160.el7.x86_64

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:user.name=root

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:user.home=/root

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:user.dir=/opt/shardingsphere-proxy

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:os.memory.free=1781MB

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:os.memory.max=1945MB

[INFO ] 2021-11-23 21:47:01.049 [main] org.apache.zookeeper.ZooKeeper - Client environment:os.memory.total=1945MB

[ERROR] 2021-11-23 21:47:01.065 [main] o.a.c.f.imps.CuratorFrameworkImpl - Background exception was not retry-able or retry gave up

java.lang.NumberFormatException: For input string: "2181?username=admin&password=123456"

	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)

	at java.lang.Integer.parseInt(Integer.java:580)

	at java.lang.Integer.parseInt(Integer.java:615)

	at org.apache.zookeeper.client.ConnectStringParser.<init>(ConnectStringParser.java:77)

	at org.apache.zookeeper.ZooKeeper.createDefaultHostProvider(ZooKeeper.java:1514)

	at org.apache.zookeeper.ZooKeeper.<init>(ZooKeeper.java:1107)

	at org.apache.zookeeper.admin.ZooKeeperAdmin.<init>(ZooKeeperAdmin.java:166)

	at org.apache.curator.utils.DefaultZookeeperFactory.newZooKeeper(DefaultZookeeperFactory.java:30)

	at org.apache.curator.framework.imps.CuratorFrameworkImpl$2.newZooKeeper(CuratorFrameworkImpl.java:210)

	at org.apache.curator.HandleHolder$1.getZooKeeper(HandleHolder.java:100)

	at org.apache.curator.HandleHolder.getZooKeeper(HandleHolder.java:48)

	at org.apache.curator.ConnectionState.reset(ConnectionState.java:195)

	at org.apache.curator.ConnectionState.start(ConnectionState.java:102)

	at org.apache.curator.CuratorZookeeperClient.start(CuratorZookeeperClient.java:214)

	at org.apache.curator.framework.imps.CuratorFrameworkImpl.start(CuratorFrameworkImpl.java:334)

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.initCuratorClient(CuratorZookeeperRepository.java:119)

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.init(CuratorZookeeperRepository.java:82)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.createClusterPersistRepository(ClusterContextManagerBuilder.java:119)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.beforeBuildContextManager(ClusterContextManagerBuilder.java:94)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.build(ClusterContextManagerBuilder.java:84)

	at org.apache.shardingsphere.proxy.initializer.BootstrapInitializer.initContext(BootstrapInitializer.java:77)

	at org.apache.shardingsphere.proxy.initializer.BootstrapInitializer.init(BootstrapInitializer.java:68)

	at org.apache.shardingsphere.proxy.Bootstrap.main(Bootstrap.java:47)

[ERROR] 2021-11-23 21:47:02.580 [main] o.a.curator.CuratorZookeeperClient - 

java.io.IOException: java.lang.NumberFormatException: For input string: "2181?username=admin&password=123456"

	at org.apache.curator.ConnectionState.close(ConnectionState.java:121)

	at org.apache.curator.CuratorZookeeperClient.close(CuratorZookeeperClient.java:243)

	at org.apache.curator.CuratorZookeeperClient.close(CuratorZookeeperClient.java:226)

	at org.apache.curator.framework.imps.CuratorFrameworkImpl.close(CuratorFrameworkImpl.java:402)

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.initCuratorClient(CuratorZookeeperRepository.java:124)

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.init(CuratorZookeeperRepository.java:82)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.createClusterPersistRepository(ClusterContextManagerBuilder.java:119)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.beforeBuildContextManager(ClusterContextManagerBuilder.java:94)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.build(ClusterContextManagerBuilder.java:84)

	at org.apache.shardingsphere.proxy.initializer.BootstrapInitializer.initContext(BootstrapInitializer.java:77)

	at org.apache.shardingsphere.proxy.initializer.BootstrapInitializer.init(BootstrapInitializer.java:68)

	at org.apache.shardingsphere.proxy.Bootstrap.main(Bootstrap.java:47)

Caused by: java.lang.NumberFormatException: For input string: "2181?username=admin&password=123456"

	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)

	at java.lang.Integer.parseInt(Integer.java:580)

	at java.lang.Integer.parseInt(Integer.java:615)

	at org.apache.zookeeper.client.ConnectStringParser.<init>(ConnectStringParser.java:77)

	at org.apache.zookeeper.ZooKeeper.createDefaultHostProvider(ZooKeeper.java:1514)

	at org.apache.zookeeper.ZooKeeper.<init>(ZooKeeper.java:1107)

	at org.apache.zookeeper.admin.ZooKeeperAdmin.<init>(ZooKeeperAdmin.java:166)

	at org.apache.curator.utils.DefaultZookeeperFactory.newZooKeeper(DefaultZookeeperFactory.java:30)

	at org.apache.curator.framework.imps.CuratorFrameworkImpl$2.newZooKeeper(CuratorFrameworkImpl.java:210)

	at org.apache.curator.HandleHolder$1.getZooKeeper(HandleHolder.java:100)

	at org.apache.curator.HandleHolder.internalClose(HandleHolder.java:115)

	at org.apache.curator.HandleHolder.closeAndClear(HandleHolder.java:78)

	at org.apache.curator.ConnectionState.close(ConnectionState.java:116)

	... 11 common frames omitted

Exception in thread "main" org.apache.shardingsphere.mode.repository.cluster.ClusterPersistRepositoryException: org.apache.zookeeper.KeeperException$OperationTimeoutException: KeeperErrorCode = OperationTimeout

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.handler.CuratorZookeeperExceptionHandler.handleException(CuratorZookeeperExceptionHandler.java:51)

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.initCuratorClient(CuratorZookeeperRepository.java:128)

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.init(CuratorZookeeperRepository.java:82)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.createClusterPersistRepository(ClusterContextManagerBuilder.java:119)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.beforeBuildContextManager(ClusterContextManagerBuilder.java:94)

	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.build(ClusterContextManagerBuilder.java:84)

	at org.apache.shardingsphere.proxy.initializer.BootstrapInitializer.initContext(BootstrapInitializer.java:77)

	at org.apache.shardingsphere.proxy.initializer.BootstrapInitializer.init(BootstrapInitializer.java:68)

	at org.apache.shardingsphere.proxy.Bootstrap.main(Bootstrap.java:47)

Caused by: org.apache.zookeeper.KeeperException$OperationTimeoutException: KeeperErrorCode = OperationTimeout

	at org.apache.shardingsphere.mode.repository.cluster.zookeeper.CuratorZookeeperRepository.initCuratorClient(CuratorZookeeperRepository.java:125)

	... 7 more
```
